app/src/main/java/com/babylon/wallet/android/presentation/transaction/analysis/TransactionAnalysisDelegate.kt

```diff
@@ -45,11 +48,15 @@ class TransactionAnalysisDelegate @Inject constructor(
     /**
      * Runs analysis on each transaction type received. Resolves the preview type and returns the signers involved.
      */
-    suspend fun analyse(): Result<Analysis> = when (data.value.request.transactionType) {
+    suspend fun analyse(): Result<Analysis> = when (val txType = data.value.request.transactionType) {
         is TransactionType.PreAuthorized -> analysePreAuthTransaction(
             manifestData = data.value.request.unvalidatedManifestData
         )
 
+        is TransactionType.DeleteAccount -> analyseDeleteAccountTransaction(
+            accountAddress = txType.accountAddress
+        )
```

Comment:
I guess this will be removed right? And the processor will handle it?

Reply:
It is just for testing purposes?

Reply:
Hmm, why?